data/styles/vehicle/include/Roads.mapcss

```diff
@@ -658,6 +658,21 @@ line|z18[highway=construction],
 line|z19-[highway=construction],
 {width: 6;dashes: 18,6.2;}
 
+/* 8.7 Footway & Steps 18-22 ZOOM */
+
+/* [highway=footway] should be defined before [highway=footway][sac_scale] in 8.6. */
+line|z18-[highway=footway],
```

Comment:
It can be useful for car drivers to see crosswalks only at detailed zoom levels.

Reply:
z18 is pretty much detailed, isn't it?